examples/resources/grafana_slo/resource_ratio.tf

```diff
@@ -0,0 +1,43 @@
+resource "grafana_slo" "ratio" {
+  name        = "Terraform Testing - Ratio Query"
+  description = "Terraform Description - Ratio Query"
+  query {
+    ratio {
+      success_metric  = "kubelet_http_requests_total{status!~\"5..\"}"
+      total_metric    = "kubelet_http_requests_total"
+      group_by_labels = ["job", "instance"]
+    }
+    type = "ratio"
+  }
+  objectives {
+    value  = 0.995
+    window = "30d"
+  }
+  label {
+    key   = "slo"
+    value = "terraform"
+  }
+  alerting {
+    fastburn {
+      annotation {
+        key   = "name"
+        value = "SLO Burn Rate Very High"
+      }
+      annotation {
+        key   = "name"
```

Comment:
shouldn't this be `key = "description", so that the generated alert has both a name, and a description annotation?